### ISSUE TITLE
How to use Nix 2.14?

### ISSUE THREAD

**Issue by srid** [nixbuild/nix-quick-install-action#28]
I tried:

```yaml
      - uses: nixbuild/nix-quick-install-action@master
        with:
          # We use newer Nix for:
          # - https://github.com/NixOS/nix/issues/7263
          # - https://github.com/NixOS/nix/issues/7026
          nix_version: 2.14.1
          nix_conf: |
            experimental-features = nix-command flakes
            access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

But this throws:

```
zstd: /*stdin*\: unsupported format 
Error: Process completed with exit code 1.
```

https://github.com/srid/haskell-flake/actions/runs/4411324946/jobs/7729732345

**Comment by rickynils** [comment]
2.14.1 will not be available until next release. On each release all currently supported Nix versions will be packaged and deployed so that the action can use it during installation. This way we can offer a fully deterministic behavior.

However, I'm going to start work on a release now, since you reminded me :)

Also, that error message looks awful, maybe we can do something about that.

**Comment by rickynils** [comment]
[v22](https://github.com/nixbuild/nix-quick-install-action/releases/tag/v22) has now been released, using Nix 2.14.1 by default.

**Comment by srid** [comment]
Thanks! That worked: https://github.com/srid/haskell-flake/pull/125
